run.py

```diff
@@ -1,28 +1,34 @@
-from scripts.data import load_data
-from scripts.data import write_annotate
+from scripts.data import load
+from scripts.data import write
 
 import scripts.preprocess.preprocess as preproc
 from collections import ChainMap
 
-# get user_params and raw
-user_params = load_data.load_param("user_params.json")
-eeg_obj = load_data.load_raw(
-    "CMI/rawdata/",
-    "NDARAB793GL3",
-    "01",
-    "ContrastChangeBlock1",
-    "01",
-    "eeg"
-)
-
-# specify preprocess steps
+user_params = load.load_param("user_params.json")
+
+# get preprocessing steps
 preprocess_steps = user_params["preprocess"]
+# get collection of subjects from user_params
+root, subjects, ch_type = load.init_data(user_params)
+
+# TODO: Should analysis be run on select tasks, runs, sessions for subjects?
+for subj in subjects:
+    subject_files = load.load_subject_files(root, subj, ch_type)
+    # get only eeg data files
+    subject_files = [s for s in subject_files if s.suffix == ch_type]
```

Comment:
When we do a bids_path.match(), which is in load_subject_files, does it not already search for paths that have the same data type?

Reply:
It looks like the `match()` function will also load files such as 

> sub-NDARAB793GL3_ses-01_task-ContrastChangeBlock1_run-01_eeg.vhdr

I'm guessing the `datatype` parameter of `mne_bids.BIDSPath` checks for "_eeg" within the file name as opposed to the file type, so I included an additional step to remove files like these.

There's probably a better way to do this within `load_subject_files`, will look into it.

Reply:
```suggestion
```